CHANGELOG.md

```diff
@@ -9,6 +9,16 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.4.1 - 2019-02-27
+
+### Fixed
+
+- Fix issues with IPsec support, which was broken in 0.4.0:
+   * check and update (if needed) the type of the default tunnel port (tun0) after Agent starts
```

Comment:
This one can be treated as a separate issue.

Reply:
Done